portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessDeliveryAPI/ExposeErcFromAssetLibraryForComments.testcase

```diff
@@ -88,6 +167,112 @@ definition {
 		}
 	}
 
+	@disable-webdriver = "true"
+	@priority = "4"
+	test CanGetChildCommentByExternalReferenceCode {
+		property portal.acceptance = "true";
+
+		task ("And Given with POST request I create a comment with a custom external reference code") {
+			var blogPostingId = BlogPostingAPI.getBlogPostingsIdList();
+
+			var response = CommentAPI.createCommentInBlogPosting(
+				blogPostingId = "${blogPostingId}",
+				commentText = "CommentOne",
+				externalReferenceCode = "ercComment");
+		}
+
+		task ("When create a child comment with a custom external reference code") {
+			var commentId = JSONUtil.getWithJSONPath("${response}", "$.id");
+
+			var response2 = CommentAPI.createCommentInBlogPosting(
+				commentId = "${commentId}",
+				commentText = "ChildComment",
+				externalReferenceCode = "ERC_ChildComment");
+		}
+
+		task ("When I make a GET request by external reference code of the child comment") {
+			var commentText = CommentAPI.getFieldValueOfBlogPostingComments(
+				blogPostingChildCommentERC = "ERC_ChildComment",
+				blogPostingCommentERC = "ercComment",
+				fieldName = "text");
+		}
+
+		task ("Then I receive a correct body response") {
+			TestUtils.assertEquals(
+				actual = "${commentText}",
+				expected = "<p>ChildComment</p>");
+		}
+	}
+
+	@disable-webdriver = "true"
+	@priority = "4"
+	test CanGetCommentByExternalReferenceCode {
+		property portal.acceptance = "true";
+
+		task ("When with POST request I create a comment with a custom external reference code") {
+			var blogPostingId = BlogPostingAPI.getBlogPostingsIdList();
+
+			CommentAPI.createCommentInBlogPosting(
+				blogPostingId = "${blogPostingId}",
+				commentText = "CommentOne",
+				externalReferenceCode = "ercComment");
+		}
+
+		task ("When I make a GET request by external reference code") {
+			task ("Get Blog Posting ID") {
```

Comment:
I think we only use GWT in task(), and we can getField with response, for example:
```
CommentAPI.getFieldValueOfBlogPostingComments(
				responseToParse = "${response}",
				fieldName = "text");
```